plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PolylineAnnotation.kt

```diff
@@ -389,20 +391,41 @@ class PolylineAnnotation(
     moveDistancesObject: MoveDistancesObject
   ): LineString? {
     val points = geometry.coordinates()
-      .map {
-        mapCameraManagerDelegate.pixelForCoordinate(it)
-      }.map {
-        mapCameraManagerDelegate.coordinateForPixel(
-          ScreenCoordinate(
-            it.x - moveDistancesObject.distanceXSinceLast,
-            it.y - moveDistancesObject.distanceYSinceLast
-          )
-        )
-      }
-    if (points.any { it.latitude() > MAX_MERCATOR_LATITUDE || it.latitude() < MIN_MERCATOR_LATITUDE }) {
+    if (points.size == 0) return null
+
+    var sumLat = 0.0
+    var sumLong = 0.0
+    points.forEach { point ->
+      sumLat += point.latitude()
+      sumLong += point.longitude()
+    }
+
+    val centerPoint = Point.fromLngLat(sumLong / points.size, sumLat / points.size)
```

Comment:
nit : center can be calculated with `average` extension eg.
`points.map { it.latitude() }.average()` / `points.map { it.longitude() }.average()`